src/main/java/org/restcomm/sdk/domain/CallDirection.java

```diff
@@ -0,0 +1,34 @@
+package org.restcomm.sdk.domain;
+
+/**
+ * Represents the direction of a call.
+ * <p>
+ * Possible values are inbound, outbound-api, and outbound-dial.
+ *
+ * @author Henrique Rosa (henrique.rosa@telestax.com) created on 21/05/2018
+ * @see <a href=https://www.restcomm.com/docs/connect/api/calls-api.html#resource-properties>Call API</a>
+ */
+public enum CallDirection {
```

Comment:
Why do we need this class?

Reply:
Can you provide an example or unit test?

Reply:
I do assertions based on call status for synthetic monitoring. It's better to rely on enum rather than hardcode string values everywhere.